javascript/src/test/components/Courses/CourseList.test.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { render } from "@testing-library/react";
+// import CourseList from "main/components/Courses/CourseList";
+import CourseList from "main/components/Footer/AppFooter";
+import {BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+import { useHistory } from "react-router-dom";
+jest.mock("react-router-dom", () => ({
+  useHistory: jest.fn(), // and this one too
+  Link: jest.fn() // and this one too
```

Comment:
These comments don't make sense and don't seem to serve any purpose, can we remove them?

Reply:
I will remove this line in the next commit